SharpDenizenTools/MetaHandlers/MetaDocs.cs

```diff
@@ -115,6 +98,23 @@ public class MetaDocs
         /// <summary>Set of raw adjustable keys.</summary>
         public HashSet<string> RawAdjustables = [];
 
+        /// <summary>Creates a new instance of <see cref="MetaDocs"/> and registers its <see cref="MetaTypeData{T}"/>.</summary>
+        public MetaDocs()
+        {
+            // Extensions explicitly first
+            IMetaTypeData.Register(this, META_TYPE_EXTENSION, Extensions, () => new MetaExtension());
```

Comment:
Why is this a static method on the interface when its purpose is affecting instance data on the MetaDocs instance?

Reply:
Moved to `MetaDocs`